apps/wallet-connect/src/hooks/useWalletConnectV1.tsx

```diff
@@ -62,9 +62,13 @@ const useWalletConnectV1 = (
         trackEvent(NEW_SESSION_ACTION, WALLET_CONNECT_VERSION_1, wcConnector.peerMeta)
 
         setWcClientData(payload.params[0].peerMeta)
+
+        // HACK: Always enable off-chain msgs
+        web3Provider.send('safe_setSettings', [{ offChainSigning: true }])
```

Comment:
@iamacook how can we make this work for OpenSea w/o this hack?
Basically, not just OpenSea, but any app that supports EIP-1271 will work out of the box with 1/X Safe.

Reply:
For future reference: after discussing this, we will look into defaulting OpenSea to off-chain for 1/n Safes.